db/schema.rb

```diff
@@ -33,6 +33,39 @@
   add_index "authentications", ["uid", "provider"], :name => "index_authentications_on_uid_and_provider", :unique => true
   add_index "authentications", ["user_id", "provider"], :name => "index_authentications_on_user_id_and_provider", :unique => true
 
+  create_table "c_rater_feedback_items", :force => true do |t|
```

Comment:
Okay. I'll see, I feel I'll start with 2 then.

As we need to store somewhere 'feedback on feedback' and mentioned multiple choice feedback, this linking model will be necessary anyway. In fact this model will represent everything we need for the researcher report too. Remaining work would be Portal-LARA communication and .xls file generation.